libs/form/src/next/select-rich/vl-select-rich.component.ts

```diff
@@ -151,9 +151,12 @@ export class VlSelectRichComponent extends FormControl {
             }
         }
 
-        if (changedProperties.has('deletable')) {
-            this.choices.config.removeItemButton = this.deletable;
-            this.choices.config.removeItems = this.deletable;
+        if (changedProperties.has('not-deletable')) {
+            this.choices.config.removeItemButton = false;
+            this.choices.config.removeItems = false;
+        } else {
+            this.choices.config.removeItemButton = true;
+            this.choices.config.removeItems = true;
```

Comment:
Zeker niet verkeerd, maar is er een bepaalde reden om te ontdubbelen?

```ts
if (changedProperties.has('not-deletable')) {
     this.choices.config.removeItemButton = !this.notDeletable;
     this.choices.config.removeItems = !this.notDeletable;
}
```

Eventueel aanpassen naar eigen inzicht.

Reply:
Dat ze perfect in sync zijn kan ik niet bevestigen; maar wanneer je de `changedProperties` uitleest voor `non-deletable` zal tegen dan `this.nonDeletable` correct op `true`/`false` staan dan ben ik wel zeker.
Het mag gerust zo blijven hoor.

Reply:
we hadden beide ongelijk, het moet dit zijn:

if (changedProperties.has('notDeletable')) {
    this.choices.config.removeItemButton = !this.notDeletable;
    this.choices.config.removeItems = !this.notDeletable;
}

Reply:
Juist :)
Dit (`if (changedProperties.has('not-deletable')) {`) was me niet eens opgevallen maar heb idd ook verkeerd overgenomen. Had het vooral over het ontdubbelen